### ISSUE TITLE
Open parenthesis of method invocations should be placed on same line as invoked methods

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#523]
To ease reading, we should file an issue when open parenthesis of method invocations are not placed on the same line as the invoked method itself.

Following code should be reported:
```c#
    public void DoSomething()
    {
        GC.Collect
                  ();
    }
```

Following code should NOT be reported:
```c#
    public void DoSomething()
    {
        GC.Collect();
    }
```
